src/Http/Http.Results/src/Results.cs

```diff
@@ -218,9 +219,42 @@ public static IResult Json(object? data, JsonSerializerOptions? options = null,
             EntityTag = entityTag,
         };
 
+    /// <summary>
+    /// Writes the byte-array content to the response.
+    /// <para>
+    /// This supports range requests (<see cref="StatusCodes.Status206PartialContent"/> or
+    /// <see cref="StatusCodes.Status416RangeNotSatisfiable"/> if the range is not satisfiable).
+    /// </para>
+    /// <para>
+    /// This API is an alias for <see cref="File(byte[], string, string?, bool, DateTimeOffset?, EntityTagHeaderValue?)"/>.</para>
+    /// </summary>
+    /// <param name="contents">The file contents.</param>
+    /// <param name="contentType">The Content-Type of the file.</param>
+    /// <param name="fileDownloadName">The suggested file name.</param>
+    /// <param name="enableRangeProcessing">Set to <c>true</c> to enable range requests processing.</param>
+    /// <param name="lastModified">The <see cref="DateTimeOffset"/> of when the file was last modified.</param>
+    /// <param name="entityTag">The <see cref="EntityTagHeaderValue"/> associated with the file.</param>
+    /// <returns>The created <see cref="IResult"/> for the response.</returns>
+#pragma warning disable RS0026 // Do not add multiple public overloads with optional parameters
```

Comment:
Should we just suppress this warning globally? We've pretty much ignored it every time we've encountered it.

Reply:
We should improve the rule in the analyzer to ignore methods that don't have the set of required parameters.